terraform/README.md

```diff
@@ -1,8 +1,101 @@
 # Terraform Scripts for provisioning AWS
 
-## Initial setup
+## Overview
 
-This setup needs to be run only once per AWS account.
+This repository contains one folder for each AWS account whose infrastructure is managed through terraform. Currently,
+it supports:
+* [`development`](/development) - The Simple AWS Dev account, managing Sandbox, QA, and Security environments
+* [`bangladesh`](/bangladesh) - The Simple Bangladesh account, managing Bangladesh Demo and Bangaldesh Production
+  environments
+
+If you want to set up a new AWS account, go to [Setting Up A New AWS Account](#setting-up-a-new-aws-account). Otherwise
+go to [Getting Started](#getting-started).
```

Comment:
We also need to copy the aws credentials to `~/.aws`, this is mentioned in `Setting Up A New AWS Account`. We should maybe add a step about it here.

Also could you pl check if the credentials file is present in 1pass? Thx.

Reply:
We can use our individual IAM creds for this, right?

Reply:
This is the AWS CLI key pair for the account, I think these are just one per account and not related to IAM. So there's one for BD, one in dev etc.

Reply:
I see the dev root creds in there, not BD. 🤔

Reply:
Actually hold up, I see the root login creds for accounts. That should be sufficient to grab em 👍